### ISSUE TITLE
e-Manifest NPM package source code missing

### ISSUE THREAD

**Issue by megankirkbride** [USEPA/e-manifest#1622]
Package: https://www.npmjs.com/package/emanifest

When I try to install version 0.3.0 of the package, the source files are not included. Only package.json, 
LICENSE, and README.md are included. https://registry.npmjs.org/emanifest/-/emanifest-0.3.0.tgz

I had to downgrade to version 0.0.22 to get any of the source files included in the package.

**Comment by dpgraham4401** [comment]
Hey @megankirkbride, thanks for submitting the ticket. 

And thank you for your patience, demand for the emanifest JS package has been realtively small so it has not recieved as much TLC.

**Comment by dpgraham4401** [comment]
@megankirkbride I've published 0.3.1 which addresses these issues. Those changes are reflected in #1624. 

However, the team decided that we can't promise long-term support for the node package given the relatively small demand. If you need this packages functionality for anything more than a one-off script, you can always copy the source and inline the dependency. We do welcome PRs if you'd like to contribute your changes back upstream.

**Comment by megankirkbride** [comment]
@dpgraham4401 Thanks for the quick turnaround! I tested the new version and it did work, although I noticed a few more issues with the package. I'll detail those here in case you do want to fix them, but given the lack of use of the package, I understand if they won't be prioritized. I likely will just call the API endpoint directly rather than use the package, given these issues.

- `autoAuth` not working on the newClient. `Object literal may only specify known properties, but 'autoAuth' does not exist in type 'RcraClientConfig'. Did you mean to write 'authAuth'?` I assume this parameter was renamed at some point and this just fell through the cracks. 
- The `RCRAINFO_PROD` url is defined incorrectly in the package. It's missing `rcrainfoprod`. Basically the same issue I described here can be replicated with the package: https://github.com/USEPA/e-manifest/issues/1623 

Again, thanks for taking a look, but no worries if these don't get fixed.
